config/jobs/kubernetes/community/community-presubmit.yaml

```diff
@@ -1,22 +1,20 @@
 presubmits:
   kubernetes/community:
   - name: pull-community-verify
+    decorate: true
     branches:
-    - master
+    - ^master$
```

Comment:
I don't think this makes a difference, does it?

Reply:
these are regexes (there was a time when they weren't, I think), so this is to prevent running against `master-of-the-universe` and such